surefire-provider/src/test/java/org/arquillian/smart/testing/surefire/provider/LoaderVersionExtractorSurefireJUnit5Test.java

```diff
@@ -0,0 +1,50 @@
+package org.arquillian.smart.testing.surefire.provider;
+
+import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.Arrays;
+import java.util.Collection;
+import org.jboss.shrinkwrap.resolver.api.maven.Maven;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(Parameterized.class)
+public class LoaderVersionExtractorSurefireJUnit5Test {
```

Comment:
same as the previous one

Reply:
If you check the project before this PR you'll notice that each component we detect has its own LoaderVersion test class, one for TestNG, other for JUnit component and so on. So I am just following the same pattern as before. We can change, but in this case this was implemented following the same approach.

Reply:
Yeah, right... It just looks weird that it is the very same test with one parameter. 
But I agree.